### PR TITLE
KL-292 Create K8s manifest for Chief Onboarding

### PR DESCRIPTION
**Pull request by geodome**

### CODE REVIEW

**Comment by telepenin** [comment]
Could you use [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) for skeleton of file structure?

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+version: '3'
```
Comment (ynnt) [inline docker-compose.yml:1]: This docker-compose file will not work

File: manifest/nuxt-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      annotations:
+        kompose.cmd: /snap/kompose/19/kompose-linux-amd64 convert
+        kompose.version: 1.21.0 (992df58d8)
+      creationTimestamp: null
+      labels:
+        io.kompose.network/global: "true"
+        io.kompose.service: nuxt
+    spec:
+      containers:
+      - args:
```
Comment (ynnt) [inline manifest/nuxt-deployment.yaml:29]: The container will not start

File: manifest/web-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+          value: http://0.0.0.0:3000
+        - name: DATABASE_URL
+          value: postgres://postgres:postgres@db:5432/postgres
+        - name: DEBUG
+          value: "True"
+        - name: DEFAULT_FROM_EMAIL
+          value: hello@chiefonboarding.com
+        - name: SECRET_KEY
+          value: somethingsupersecret
+        image: web
```
Comment (ynnt) [inline manifest/web-deployment.yaml:44]: How will it work?

**Review by ynnt — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (geodome) [reply docker-compose.yml:1]: This docker-compose file is not part of the k8s manifest. It originated from the chief-onboarding repo.

**Comment by geodome** [comment]
@ynnt 

I have updated the manifests.
- chiefonboarding-app.yaml defines the deployment manifest
- chiefonboarding-service.yaml defines the service manifest
- configmap.yaml contains the Caddyfile for the caddy proxy used by the chiefonboarding-app.
- chiefonboarding-db.yaml defines the deployment manifest for the postgres databased used by the chiefonboarding app.
- chiefonboarding-service.yaml defines the service manifest for the postgres db.

**Review by ynnt — APPROVED** [review]

**Pull request merged**